pkg/integrations/v4/logs/cfg.go

```diff
@@ -170,6 +170,7 @@ type FBCfgInput struct {
 	DB                    string
 	Path                  string // plugin: tail
 	BufferMaxSize         string // plugin: tail
+	Path_Key	      string // plugin: tail
```

Comment:
```suggestion
	PathKey	      string // plugin: tail
```

Reply:
Good point. You can name the internal variable in PascalCase, and then just use `Path_Key` when applying the template. Nice catch!